lib/matplotlib/axes/_axes.py

```diff
@@ -7464,7 +7464,7 @@ def specgram(self, x, NFFT=None, Fs=None, Fc=None, detrend=None,
 
         **kwargs
             Additional keyword arguments are passed on to `~.axes.Axes.imshow`
-            which makes the specgram image.
+            which makes the specgram image. The origin kwarg is ignored.
```

Comment:
```suggestion
            which makes the specgram image. The origin keyword argument
            is not supported.
```
"not supported": see above for raising instead of ignoring.

We write out "keyword argument" instead of "kwarg" in the docs. While kwarg is a common abbreviation in the python community, it's slang and may not be understood by novices. - This should also be changed in other parts of the PR.

Reply:
Thank you for the suggestions! Should I also remove kwarg from commit messages and tests, or just the public-facing docs?

Reply:
Public-facing docs is enough.